lib/absinthe/phase/schema/type_imports.ex

```diff
@@ -37,13 +37,14 @@ defmodule Absinthe.Phase.Schema.TypeImports do
 
     types = Enum.reject(other_def.type_definitions, &(&1.identifier in rejections))
 
-    case Keyword.fetch(opts, :only) do
-      {:ok, selections} ->
-        Enum.filter(types, &(&1.identifier in selections))
-
-      _ ->
-        types
-    end
+    types =
+      case Keyword.fetch(opts, :only) do
```

Comment:
No test covered this feature? It seems like this one was covering it:

https://github.com/absinthe-graphql/absinthe/blob/master/test/absinthe/schema/notation/experimental/import_fields_test.exs#L100

Reply:
This test https://github.com/absinthe-graphql/absinthe/blob/master/lib/absinthe/phase/schema/type_imports.ex covers the :only/:except logic for type imports, but the test only asserts that :only/:except is set correctly, not whether the actual import happens. I've added assertions that should check this.

Reply:
Also added some docs for this functionality and added import_types/2 to locals_without_parens

Reply:
Great!